roles/setup_replication/tasks/rm_replication.yml

```diff
@@ -50,9 +68,54 @@
   become: true
   when: pg_wal|length > 0
 
+- name: Remove postgres default data directory if exists
```

Comment:
I suggest adding another flag for the removal of the data directory and pg_wal directory, as they contain user data and it is recommended for users to use this flag when deleting these directories. What do you think?

Reply:
I will add a `force_wal` and `force_data` flag

Reply:
Should this be added to the tasks removing `pg_default_data` and `pg_local_wal_archive_dir`?

Reply:
The directory is not for WAL archives, but rather for the actual WAL files location in Postgres. The archive location can be configured using archive_command or archive_library, as described in https://www.postgresql.org/docs/current/continuous-archiving.html. As we are currently utilizing the pg_wal and pg_data variables for a user-defined location, let's create new variables with the same name, but prefix with either `rm or `force_rm`